modules/@apostrophecms/form-widget/index.js

```diff
@@ -32,5 +34,48 @@ module.exports = {
         return _super(req, widgets);
       }
     };
+  },
+  apiRoutes(self) {
+    return {
+      post: {
+        upload: [
```

Comment:
This route is almost completely taken from the attachment route. If that route function was stored in an accessible way I could have used it directly. I copied since it's not super complex and didn't see the need for the additional PR. I'd be fine doing that if needed.